tests/gis_tests/test_spatialrefsys.py

```diff
@@ -120,3 +120,26 @@ def test_add_entry(self):
         self.assertTrue(
             SpatialRefSys.get_spheroid(srs.wkt).startswith('SPHEROID[')
         )
+
+    def test_srid_info_without_gdal(self):
+        attrs = [
+            # Could be retrieved without GDAL, but value differs.
+            # 'ellipsoid',
```

Comment:
It seems that ellipsoid value could be easily fixed, but I'm not sure we need it.